### PR TITLE
convert to OTP 22.1.8 socket.erl

### PR DESCRIPTION
**Pull request by RoadRunnr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,9 +92,13 @@ ERLANG Version Support
 
 All minor version of the current major release and the highest minor version of the
 previous major release will be supported.
-At the moment this means OTP 21.3 and OTP 22.0 are supported. OTP versions
-greater than 20.1.7 could work, but are not tested or guaranteed to work. When in doubt
-check the `otp_release` section in [.travis.yml](.travis.yml) for tested versions.
+As an exception to the rule state in the previous paragraph is currently the minimum
+required version OTP 22.1.8, the latest released version at the time of writing. The
+reason is the move to the office OTP socket.erl module and the bug that has exposed
```
Comment (mgumz) [inline README.md:97]: office -> official

**Comment by mgumz** [comment]
lgtm, like a 1:1 drop-in-replacement.

minor cosmetics in the documentation, otherwise ok

File: README.md (hunk 1 above)
Comment (RoadRunnr) [reply README.md:97]: the wording of this whole paragraph sounds strange

**Comment by mgumz** [comment]
do you want to rename the commit to not include the typo?

**Comment by RoadRunnr** [comment]
> do you want to rename the commit to not include the typo?

done

**Pull request merged**